clients/src/main/java/org/apache/kafka/clients/admin/RemoveMembersFromConsumerGroupResult.java

```diff
@@ -66,6 +79,9 @@
      * Returns the selected member future.
      */
     public KafkaFuture<Void> memberResult(MemberToRemove member) {
+        if (removeAll()) {
+            throw new IllegalArgumentException("The method: memberResult is not applicable in 'removeAll' mode");
```

Comment:
Why that? I understand that we expect that users don't know the memberId if the so a "remove all"; however, I don't see why we need to disallow this call? Can you elaborate?

Reply:
Since in the `removeAll` scenario, we don't save the members to be deleted in `RemoveMembersFromConsumerGroupResult`,  so I think calling `memberResult` doesn't seem applicative.

Reply:
I see. Makes sense.